src/Listener/ApiPaginationListener.php

```diff
@@ -58,13 +58,19 @@ public function beforeRender(Event $event)
             return;
         }
 
+        $paginationLimit = $pagination['limit'];
+        if ($pagination['limit'] == null) {
+            $paginationLimit = $controller->paginate['limit'];
```

Comment:
This will generate a notice if `paginate['limit']` is not set, so need a check for that too.